modules/test/playwright/pages/calendar-web/CalendarWidgetPage.ts

```diff
@@ -5,14 +5,20 @@
 
 import {Locator, Page} from '@playwright/test';
 
+import {waitForSuccessAlert} from '../../utils/waitForSuccessAlert';
+import {ModalRecurrencePage} from './ModalRecurrencePage';
+
 export class CalendarWidgetPage {
 	readonly addEventButton: Locator;
 	readonly allDayCheckbox: Locator;
 	readonly calendarWidget: Locator;
 	readonly closeConfigurationButton: Locator;
 	readonly configurationMenuItem: Locator;
 	readonly endTime: Locator;
+	modalRecurrencePage: ModalRecurrencePage;
```

Comment:
This can be assigned in constructor, thus it is readonly.

Reply:
Hey @cgmonte,

I encountered some errors while trying to implement the change. Specifically, when calling the CalendarWidgetPage constructor, the elements of ModalRecurrencePage haven't rendered yet, causing locator errors. Am I missing something here?